python/tidy.py

```diff
@@ -103,12 +103,19 @@ def check_whitespace(idx, line):
 
 def check_by_line(file_name, contents):
     lines = contents.splitlines(True)
+    # import pdb; pdb.set_trace()
     for idx, line in enumerate(lines):
-        errors = itertools.chain(
-            check_length(idx, line),
-            check_whitespace(idx, line),
-            check_whatwg_url(idx, line),
-        )
+        if file_name.endswith(".lock"):
```

Comment:
Instead of doing this conditional here, can you add it in to the `check_length` function like [we do here](https://github.com/servo/servo/blob/83972196600f04e817ddb53fda18142778905307/python/tidy.py#L65-L66)? That way each linting function determines which files it can handle instead of handling it from a global level